README.md

```diff
@@ -24,6 +24,12 @@ Kelp is built to:
 
 To learn more about the Stellar protocol check out [Stellar Videos on Coinbase Earn][stellar coinbase earn], or [this video about the Stellar DEX created by Lumenauts][sdex explainer video], or read more about it on the [Stellar Website][intro to stellar].
 
+## Be Smart and Go Slow
+
+**Important Disclaimer: Be Smart and Go Slow.** Whenever you trade on Stellar, you are trading with volatile assets, in volatile markets, and you risk losing money. Kelp is an experimental software that contains bugs. Use Kelp at your own risk. There is no guarantee you'll make a profit from using our bots or strategies. In fact, if you set bad parameters or market conditions change, Kelp might help you lose money very fast. So be smart and go slow.
+
+Your use of Kelp is governed by the Apache 2.0 open-source license. Please note that SDF’s interactions with you are governed by the SDF [Terms of Service](https://www.stellar.org/terms-of-service) and [Privacy Policy](https://www.stellar.org/privacy-policy).
```

Comment:
Can you move the two links at the bottom? only images have inline links
The links at the end of this file are ordered based on where they appear.

see **announcements distribution list** in the below paragraph as an example.